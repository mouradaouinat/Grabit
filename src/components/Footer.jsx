import React from 'react'
import styles from '../styles/Footer.module.css'

const Footer = () => {
  return (
    <footer className={styles.footer}>
      <span className={styles.legal}>&copy; 2019 Grabit</span>{' '}
      <ul className={styles.list}>
        <li className={styles['list-item']}>Terms</li>
        <li className={styles['list-item']}>Privacy Policy</li>
      </ul>
    </footer>
  )
}

export default Footer
