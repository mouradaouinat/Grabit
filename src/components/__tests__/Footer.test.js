import React from 'react'
import { render } from '@testing-library/react'
import Footer from '../Footer'

describe('Footer', () => {
  it('renders without crashing', () => {
    render(<Footer />)
  })

  it('displays the same content every time it renders', () => {
    const snapshot = render(<Footer />)
    expect(snapshot).toMatchSnapshot()
  })
})
