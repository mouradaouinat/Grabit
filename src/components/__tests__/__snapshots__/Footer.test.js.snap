// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer displays the same content every time it renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="footer"
      >
        <span>
          © 2019 Grabit
        </span>
         
        <ul
          class="footer__legal"
        >
          <li
            class="footer__legal-item"
          >
            Terms
          </li>
          <li
            class="footer__legal-item"
          >
            Privacy Policy
          </li>
        </ul>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer"
    >
      <span>
        © 2019 Grabit
      </span>
       
      <ul
        class="footer__legal"
      >
        <li
          class="footer__legal-item"
        >
          Terms
        </li>
        <li
          class="footer__legal-item"
        >
          Privacy Policy
        </li>
      </ul>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
